.flake8

```diff
@@ -1,4 +1,11 @@
 [flake8]
-exclude=elasticapm/utils/wrapt,build,src,tests,dist,conftest.py,setup.py
+exclude=
```

Comment:
any reason for this change? It is more readable/scannable for sure, so I'm good with merging it, just wondering if there's any other reason behind it.

Reply:
Yes, since vscode invokes flake8 with a full path on save, the excludes weren't being observed. The `/**` format seems to fix the issue and is still semantically the same.

(Splitting onto different lines was just additional readability)